Formula/pet.rb

```diff
@@ -0,0 +1,61 @@
+class Pet < Formula
+  desc "Polyhedral Extraction Tool"
+  homepage "https://pet.gforge.inria.fr/"
+  url "http://pet.gforge.inria.fr/pet-0.11.5.tar.xz"
+  sha256 "e9238a3e92cc13358cdec22d94c23740ac758f649df891f4bc3461e988e879e3"
+  license "MIT"
+
+  livecheck do
+    url :homepage
+    regex(/pet-(\d+(?:\.\d+)+b?)/i)
+  end
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "gmp"
+  depends_on "isl"
+  depends_on "libyaml"
+  depends_on "llvm"
+
+  uses_from_macos "libxml2"
+  uses_from_macos "ncurses"
+  uses_from_macos "zlib"
+
+  def install
+    site_packages = Language::Python.site_packages(Formula["python@3.9"])
+
+    # avoid using llvm's libc++
+    ENV.remove "HOMEBREW_LIBRARY_PATHS", Formula["llvm"].opt_lib
```

Comment:
Why don't we want to use LLVM's libc++ here? It seems fine to use for me given that `llvm` is a runtime dependency.

We usually want to avoid LLVM's libc++ so we can keep `llvm` as a build-only dependency, but since it isn't here it seems fine to allow linkage with LLVM's libc++.